doc/rst/usingchapel/building.rst

```diff
@@ -5,19 +5,22 @@ Building Chapel
 ===============
 
 To build the Chapel compiler, set up your environment as described in
-:ref:`chapelhome-quickstart` (or :ref:`readme-chplenv` for more
-settings), cd to ``$CHPL_HOME``, and run GNU make.
+:ref:`readme-chplenv`, cd to ``$CHPL_HOME`` (the root directory of
```

Comment:
Can this be more self contained by saying how to set up the environment for the preferred, single-locale Chapel compiler and runtime and referring them to `readme-chplenv` if they want something other than that?

Reply:
The first sections of `readme-chplenv` already talk about the preferred environment pretty directly and succinctly (which is not to say it couldn't be improved or streamlined further, but that's a separate task).  So if we were to say more here rather than redirecting, I fear we get into the case of every file trying to say everything redundantly rather than each playing its position (where I view this as being designed to be the second file you'd read after that one).

One of the ironic things about the preferred, single-locale Chapel build (e.g., `setchplenv.bash`) is that it doesn't actually set anything interesting because the `CHPL_*` variables are designed to default to reasonable values.  Specifically, it sets:
* CHPL_HOME (which might even no longer be necessary... we've removed most dependencies on it in recent years)
* PATH to include the `bin/` directory
* MANPATH to include the `man/` directory
These are called out in the `Recommended Settings` portion of `building.rst` directly and indirectly via the mention of the `setchplenv.*` scripts in the second paragraph.

Reply:
Ok.